help/dm-mention-alert-notifications.md

```diff
@@ -54,25 +54,25 @@ channels in your [Channel settings](/help/channel-notifications), and
 administrators can [restrict use of wildcard
 mentions](/help/restrict-wildcard-mentions) in large channels.
 
-## Alert words
+## Watched phrases
 
-Zulip lets you to specify **alert words or phrases** that send you a desktop
-notification whenever the alert word is included in a message. Alert words are
+Zulip lets you to specify **watched phrases** that send you a desktop
+notification whenever the watched phrase is included in a message. Watched phrases are
 case-insensitive.
 
 !!! tip ""
 
-    Alert words in messages you receive while the alert is enabled will be highlighted.
+    Watched phrases in messages you receive while the alert is enabled will be highlighted.
 
-### Add an alert word or phrase
+### Add a watched phrase
 
 {start_tabs}
 
-{settings_tab|alert-words}
+{settings_tab|watched-phrases}
 
 1. Add a word or phrase.
```

Comment:
This is out-of-date with the current UI, which has a button that opens a modal ...